applications/nrf_desktop/src/modules/config.c

```diff
@@ -142,6 +149,49 @@ static void update_config(const u8_t config_id, u8_t *data, size_t size)
 	}
 }
 
+static void settings_load_thread(void)
+{
+	LOG_INF("Settings load thread started");
+
+	int err = 0;
+	
+	err = settings_load();
+	if (err) {
+		LOG_ERR("Cannot load settings");
+	}
+	LOG_INF("Settings loaded");
```

Comment:
This should be surrounded with an else clause.

Can't load.
Loaded.

looks weird in the logs.

Reply:
corrected